shedlib/UI/UserInterface.py

```diff
@@ -1,16 +1,28 @@
-
+from shedlib.Time.Time import Time
 
 class UserInterface:
    """Basic user interface class"""
    def __init__(self) -> None:
       """Initialization method"""
+
       self.queue = []
+      """queue to show"""
 
    def update(self):
       """Main method. Update current scene"""
       pass
 
    def append_points(self, *points) -> None:
```

Comment:
I don't really like *points getting parameters. We have to do something with this. 
This way of getting parameters doesn't seem right. It may cause some problems in future.